fr-portfolio.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en-gb">
+<head>
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<link rel="stylesheet" type="text/css" href="portfolio.css">
+	<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+	<title>Freddie's Portfolio</title>
+</head>
+<body>	
+ <main>	
+  <nav class="navbar">
+   <a href="#portfolio">Portfolio</a>
+   <a href="#about">About</a>
+   <a href="#contact">Contact</a>
+  </nav>
+	
+  <section class="welcome-banner" >
+   <h1>Freddie's Portfolio</h1>
+  </section>	
+	
+  <article id="about" class="about-alt">
+	<h2> About me </h2>
+		<div class="about-me-container">
+			<figure>
+				<img src="https://via.placeholder.com/350x200">
+			</figure>
+			<div>
+				<p>Hi this is where I should put some information about myself</p>
+			</div>
+		</div>		
```

Comment:
Formatting needs a little work, try to make all children of an element have the same indentation for clarity. Don't forget the `<figcaption>` tag and `alt="stuff"` for the image. You can probably remove the `<div>` which only contains a `<p>`, if a `<div>` only has one child it can usually be removed.

Reply:
Div has been removed. I have left P tag as this will be couple sentences about myself rather than a caption for the picture.